modules/rtm/DESCRIPTION

```diff
@@ -9,12 +9,11 @@ Authors@R: c(person("Mike", "Dietze", role = c("aut"),
              person("Alexey", "Shiklomanov", role = c("aut", "cre"),
                      email = "alexey.shiklomanov@pnnl.gov"),
              person("University of Illinois, NCSA", role = c("cph")))
-Author: Alexey Shiklomanov, Shawn Serbin, Mike Dietze
-Maintainer: Alexey Shiklomanov <ashiklom@bu.edu>
 Description: This package contains functions for performing forward runs and
     inversions of radiative transfer models (RTMs). Inversions can be performed
     using maximum likelihood, or more complex hierarchical Bayesian methods.
     Underlying numerical analyses are optimized for speed using Fortran code.
+Depends: R (>= 2.10)
```

Comment:
R adds this automatically because some of the example data are not readable in older versions. I dunno, does depending on R newer than 2.1  force people to live too much on the cutting edge? 😁